### DISCUSSION TITLE
Create a generic or specific way to add different types of Cookies

### DISCUSSION THREAD

**Discussion by FrancescoMasaia** [dargmuesli/nuxt-cookie-control#52] [Ideas]
As I've seen on many premium cookie policy services like for example [Iubenda](https://www.iubenda.com), the cookie policy is divided in various different sections:

- Necessary
- Functional
- Experience
- Measurement
- Marketing

It could be interesting to implement them in a more fixed way or in a generic way extending the current code base.
I think that those 5 section could be enough for any use case.
But in case everyone needs more subdivision of their cookies we can think for a better solution as Jonas wrote me in the Issue related.

**Comment by dargmuesli** [comment]
This is being tracked in https://github.com/dargmuesli/nuxt-cookie-control/issues/216
